.github/github_token_rotation.md

```
@@ -0,0 +1,366 @@
+  "https://api.github.com/app" | jq -e . >/dev/null 2>&1 || {
+    echo "Error: Failed to verify GitHub App configuration" >&2
+    exit 1
+}
+```
+
+- **Expected Output**: JSON response containing details about the GitHub App, including id, name, and owner.
+- **Validation**: Verify that the output includes accurate GitHub App information, confirming the app’s configuration and authentication status.
+
+---
+
+### Security Best Practices
```

Markdownlint

Expected: h2; Actual: h3
Heading levels should only increment by one level at a time

(MD001, heading-increment)